src/AuthProvider.tsx

```diff
@@ -167,7 +182,7 @@ export const AuthProvider = (props: AuthProviderProps): JSX.Element => {
 
     // register to userManager events
     useEffect(() => {
-        if (!userManager) return undefined;
+        if (!userManager || !userManager.events) return undefined;
```

Comment:
this was included due to the fact that some tests would fail.
e.g: if in the configuration of the provider we would have `implementation: null`, the `userManager.events.addUserLoaded(handleUserLoaded)` would error since events is `null`